packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
           <atomic-layout-section section="results">
             <atomic-smart-snippet></atomic-smart-snippet>
             <atomic-folded-result-list image-size="small">
+              <atomic-result-template must-match-sourcetype="Slack">
+                <template>
+                  <style>
+                    div.result-root.with-sections.display-list.density-normal atomic-result-section-visual {
+                      margin-right: 16px;
```

Comment:
I recommend universally using `rem` instead of `px` for accessibility.

e.g.:
```suggestion
                      margin-right: 1rem;
```